.gitignore

```diff
@@ -19,6 +19,7 @@ erl_crash.dump
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
-/.nerves
 /build.log
 /archive.log
+/.nerves
+/doc
```

Comment:
does it matter that there is no new line at the end of this file?

Reply:
No, but it's annoying. I'll fix.

Reply:
I checked other systems that had this applied recently and it made its way there too :(

Reply:
I'll fix the rpi0 for the next release. I use that as my base for synchronizing all of the other systems.